### DISCUSSION TITLE
Disable caching locally

### DISCUSSION THREAD

**Discussion by sstottelaar** [statamic/cms#11002] [Q&A] [answered]
Hi,

Is there a global way to disable all `{{ cache for="1 hour"}}` tags during local development? I want to built in view caching, but not locally during development. Couldn't find anything about it.

Cheers!

**Comment by jacksleight** [comment, marked as answer]
There’s a `STATAMIC_CACHE_TAGS_ENABLED` env var  you can set to false. Check the `statamic.system` config file.

**Reply by sstottelaar** [reply]
Thank you! Was looking for this, works like a charm.
